PermissionsScrapper/PermissionsScrapper/ApplicationConfig.cs

```diff
@@ -0,0 +1,91 @@
+﻿// ------------------------------------------------------------------------------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace PermissionsScrapper
+{
+    /// <summary>
+    /// Describes the application configuration settings
+    /// </summary>
+    public class ApplicationConfig
+    {
+        private Dictionary<string, string> _regexReplacements;
+
+        public string Instance { get; set; } = "https://login.microsoftonline.com/{0}";
+
+        public string ApiUrl { get; set; } = "https://graph.microsoft.com/";
+
+        public string TenantId { get; set; }
+
+        public string ClientId { get; set; }
```

Comment:
please use `const` for constant values, `readonly` for values assigned only in constructor.

Reply:
Sorry, which ones are you referring to? I'm only defining the class properties in here, with the 1st two being assigned default values.

Reply:
ApiUrl looked like a constant to me.

If they are expected to be modified elsewhere, leave them as is. I just wanted to make sure that they are intended to be mutable values with default `get`s and `set`s.